src/components/map/Location.js

```diff
@@ -1,26 +1,35 @@
+import { Map } from '@styled-icons/boxicons-solid'
+import { transparentize } from 'polished'
 import PropTypes from 'prop-types'
 import styled from 'styled-components/macro'
 
+import ResetButton from '../ui/ResetButton'
+
 /**
  * Component for a location displayed on the map.
  * @param {function} onClick - The handler called when this location is clicked
  * @param {boolean} label - The optional location label that will appear underneath location icon
  */
-const LocationButton = styled.button`
-  width: 15px;
-  height: 15px;
-  padding: 0;
+const LocationButton = styled(ResetButton)`
+  position: relative;
+  width: 16px;
+  height: 16px;
   border-radius: 50%;
-  background: ${({ theme }) => theme.blue};
+  background: ${({ theme }) => transparentize(0.25, theme.blue)};
   transform: translate(-50%, -50%);
-  border: none;
-  cursor: pointer;
   box-shadow: 0 0 4px 1px rgba(0, 0, 0, 0.12);
-  opacity: 0.75;
 
   &:focus {
     outline: none;
   }
+
+  svg {
+    position: absolute;
+    transform: translate(-50%, -50%);
+    top: -12px;
+    filter: drop-shadow(0px 1px 5px rgba(0, 0, 0, 0.45));
```

Comment:
Why is this drop shadow color hard coded?

Reply:
Jeffraj said it's fine